velox/docs/functions/spark/json.rst

```diff
@@ -44,3 +44,30 @@ JSON Functions
         SELECT json_object_keys(''); -- NULL
         SELECT json_object_keys(1); -- NULL
         SELECT json_object_keys('"hello"'); -- NULL
+
+.. spark:function:: from_json(jsonString) -> [json object]
+
+    Casts a JSON string to an ARRAY, MAP, or ROW type, with the output type 
+    determined by the expression. Returns NULL, if the input string is unparsable.
```

Comment:
> with the output type determined by the expression

nit: does 'the expression' refer to 'jsonString'? Perhaps use \``jsonString\`` instead.